src/test/java/org/shikshalokam/backend/scp/TestScpProjectCRUDOperations.java

```diff
@@ -0,0 +1,302 @@
+        JSONObject requestBody = loadJsonPayload("testCreateProjectWithValidPayload");
+
+        // Generate random values for specific fields
+        requestBody.put("title", "ProjectTitle" + RandomStringUtils.randomAlphabetic(8));
+        requestBody.put("objective", "Objective" + RandomStringUtils.randomAlphabetic(20));
+        requestBody.put("keywords", "Keyword" + RandomStringUtils.randomAlphabetic(10));
+
+        // Make the API call and verify the response
+        JSONObject createRequestBody = updateProject(requestBody);
+
+        // Call updateRequest with the created requestBody
+        Response response = updateRequest(createRequestBody);
+
+        // Log the status code and response body
+        int statusCode = response.getStatusCode();
+        response.prettyPrint();
+
+        // Validate response code is 202
+        Assert.assertEquals(statusCode, 202, "Status code should be 202");
+
+        logger.info("Ended calling the projectUpdation API with valid payload.");
+    }
+
+
+    @Test(description = "Verifies the functionality of updating new project with invalid payload using random strings.")
+    public void testUpdateProjectWithInValidPayload() throws Exception {
+        logger.info("Started calling the UpdateProject API with invalid payload:");
+
+        // Load JSON payload and ensure the file exists
+        JSONObject requestBody = loadJsonPayload("testCreateProjectWithValidPayload");
+
+        // Use random strings for all values to create an invalid payload
+        requestBody.put("title", " " + RandomStringUtils.randomAlphabetic(8));
+        requestBody.put("objective", " " + RandomStringUtils.randomAlphabetic(20));
+
+        // Make the API call and verify the response
+        JSONObject createRequestBody = updateProject(requestBody);
+
+        // Call updateRequest with the created requestBody
+        Response response = updateRequest(createRequestBody);
+
+        // Log the status code and response body
+        int statusCode = response.getStatusCode();
+        response.prettyPrint();
+
+        // Validate response code is 400
+        Assert.assertEquals(statusCode, 400, "Status code should be 400 for invalid payload");
+
+        logger.info("Ended calling the projectUpdation API with invalid payload.");
+    }
+
+    @Test(description = "Verifies the functionality of getting project details with valid project id.")
+    public void testGetProjectDetailsWithValidProjectId() {
+        logger.info("Started calling the getProjectDetails API valid project ID.");
+
+        // Call updatePermission with the created requestBody
+        Response response = getProjectDetailsRequest(createdId);
+
+        // Log the status code and response body
+        response.prettyPrint();
+
+        // Validate response code
+        Assert.assertEquals(response.getStatusCode(), 200, "Status code should be 200");
+        logger.info("Ended calling the projectCreation API with valid payload.");
```

Comment:
No asserts for fetched project details, how do you make sure all data is intact while fetching :
we need lot of validations